cliconfig/loader.go

```diff
@@ -225,7 +225,7 @@ func (l Loader) setFieldValueFromCLI(fieldName string, cliName string) error {
 				} else if fieldKind == reflect.Int {
```

Comment:
[Optional] Time to switch these if..else if..else if.. chains out for a neater looking switch?

Reply:
yep, good call. i had the same thought going through the code initially